distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc

```diff
@@ -0,0 +1,52 @@
+:title: Validator Plugins on Ingest
+:type: dataManagement
+:status: published
+:parent: Validating Data
+:order: 00
+:summary: Validator plugins run on Ingest
+
+== {title}
+
+On ingest, some validator plugins are run to ensure the data being ingested will be valid
+depending on whether *Enforce errors* is selected within the admin console. Below is a list
+of the validators run against the data ingested.
+
+[NOTE]
+====
+To navigate to turn on/off *Enforce errors*:
```

Comment:
We could make this a `.Title` above the `[NOTE]` line. I also think we can cut the `To navigate to` and just make it `Enforcing Errors`

Reply:
fixed